### PR TITLE
[REG-1961] Updated Monkey Bot docs

### PR DESCRIPTION
**Pull request by vontell**
Updates the Monkey Bot documentation to account for bot segment setup (rather than using the old monobehaviour approach). Copies most of Sasha's existing docs, and adds content related to the bot segment configuration. Happy to move things around and add clarifications.

Loom video: https://www.loom.com/share/63d6d00d63f54abebbf15a71c83e7f86?sid=3202df97-a07f-4d9f-bcae-7e854fb72444

### CODE REVIEW

**Review by abeizer — COMMENTED** [review]
Before I review the actual contents, I have some questions/high-level comments:

Do we no longer support the MonoBehaviour / "standalone" method of running the monkey bot? Is it only through a bot segment now?

We don't have any of the bot sequences content filled in yet, but all of the bot actions will eventually be listed under the "Bot Sequences" sidebar. As this doc stands, people _can't_ reference it because they won't have any clue what bot sequences, segments, actions, etc. are.

If we want to keep support for Monkey Bots outside of sequences, then this page needs to talk about how the Monkey Bot works, why someone should use it, and then how to set it up without any references to bot sequences. Then, once we get to filling in the different types of segments, a "Monkey Bot Action" page will link back to this one for context and simply list out the segment configuration.

If we don't want to keep support for Monkey Bot outside of sequences, then the contents 
of this page should be exclusively under the Bot Sequences sidebar.

**Comment by addisonbgross** [comment]
After reading through this, I think I agree with @abeizer 's take. The content references Segments and configuring Bot Actions, and might be confusing for users who just want a monkey bot and have not read through the Bot Sequences stuff. 

TL;DR I think this content should be within the Bot Sequences sidebar

File: docs/core-concepts/stability-and-exploratory-testing.mdx [hunk 1]
```diff
@@ -4,6 +4,106 @@ sidebar_label: 'Stability and Exploratory Testing'
...
 # Stability and Exploratory Testing
 
-import Placeholder from '../partials/_placeholder.mdx';
+Regression Games permits more than just predefined testing scenarios. Stability and exploratory testing uses bots can be used to find bugs when you may not have a specific test case in mind.
 
-<Placeholder />
\ No newline at end of file
+## Monkey Bots
+
+<div style={{position: "relative", paddingBottom: "50.40485829959515%", height: 0}}><iframe src="https://www.loom.com/embed/8cde5bbe454c497aa74bd3afacca23cc?sid=745912d2-c555-4ef3-8a24-b2befd49c808" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{position: "absolute", top: 0, left: 0, width: "100%", height: "100%"}}></iframe></div>
```
Comment (batu) [inline docs/core-concepts/stability-and-exploratory-testing.mdx:11]: I like this video! Flows well and it demonstrates the approach. However, we can consider adding a little bit more of a flare.

I think it would have been cool if we could intentionally add a bug (maybe a simple null reference exception) and end the video with having the monkey bot finding it. Another extra step might even include going to the website dashboard and using the monkey bot recording to point out an interesting <thing> whatever that might be.

I really appreciate tutorials that, in addition to showing how to use a tool, it also teaches me how to get value out of a tool.

**Review by batu — COMMENTED** [review]
Will fully reviews based on how we want to position this page.

**Comment by vontell** [comment]
@abeizer @addisonbgross just to clarify, you say under the Bot Sequences sidebar - does that mean as a subpage of Bot Sequences, a subpage of Bot Sequences > Actions, or copy-pasted **into** the Actions page as a new section?

**Comment by abeizer** [comment]
> @abeizer @addisonbgross just to clarify, you say under the Bot Sequences sidebar - does that mean as a subpage of Bot Sequences, a subpage of Bot Sequences > Actions, or copy-pasted **into** the Actions page as a new section?

Either of the last two. We don't have any of the Bot Sequences content filled out yet so we don't know exactly where this will end up, but if this entire page is about using Monkey Bot **as a BotSegment** then it needs to appear within the context of Bot Sequences and Segments. It won't make sense to users otherwise.
